### PR TITLE
Add metric for multiplexed add(pre)chain call latency

### PR DESCRIPTION
**Pull request by Mercurrent**

### CODE REVIEW

**Comment by Mercurrent** [comment]
friendly ping

File: submission/proxy.go [hunk 1]
```diff
@@ -197,5 +202,8 @@ func (p *Proxy) AddChain(ctx context.Context, rawChain [][]byte) ([]*AssignedSCT
 	if p.dist == nil {
 		return []*AssignedSCT{}, fmt.Errorf("proxy distributor is not initialized. call Run()")
 	}
+	defer func(start time.Time) {
+		rspLatency.Observe(time.Since(start).Seconds(), "add-pre-chain")
```
Comment (RJPercival) [inline submission/proxy.go]:
```suggestion
		rspLatency.Observe(time.Since(start).Seconds(), "add-chain")
```

File: submission/proxy.go [hunk 2]
```diff
@@ -44,11 +44,13 @@ const (
 var (
 	proxyOnce      sync.Once
 	logListUpdates monitoring.Counter
+	rspLatency     monitoring.Histogram // ep => value
 )
 
 // proxyInitMetrics initializes all the exported metrics.
 func proxyInitMetrics(mf monitoring.MetricFactory) {
 	logListUpdates = mf.NewCounter("log_list_updates", "Number of Log-list updates")
+	rspLatency = mf.NewHistogram("http_latency", "Latency of responses in seconds", "ep")
```
Comment (RJPercival) [inline submission/proxy.go]: What does this metric tell you that `http_log_latency` doesn't?

Comment (Mercurrent) [reply submission/proxy.go]: It tells you the response latency for multiplexed calls (according to policy). Those include several submission calls and possible additional delay if some submissions were not successful.

Comment (RJPercival) [reply submission/proxy.go]: It might be helpful to put something to that effect in the description string.

**Pull request merged**